src/ApiPort/ApiPort.GUI/README.md

```diff
@@ -22,11 +22,11 @@ Use the following instructions to run Portability Analyzer.
 7.	By pressing the **Export** button you can export the analysis to your computer in the desired format. 
 
 
-###Troubleshooting
+### Troubleshooting
 
-If your project is not built, an error warning will pop up to remind you to build your project so that the .exe file can be found by the analyzer.
+If your project is not built, an error warning will pop up to remind you to build your project so that the .exe file can be found by the analyzer. In addition, if your project is not using PackageReference for your NuGet packages, a warning will appear. Having your NuGet packages in ProjectReference format is not neccesary in order to use the analyzer, however if you wish to convert your project to .NET Core you will need to migrate your NuGet packages to PackageReference. 
```

Comment:
We should link to more info on PackageReference and how to migrate to PackageReference.